docker.production/run_webapp.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#user pontoon stuff --> get enc vars
+(echo ">>> running as user: " && whoami)
+if [ ! -z "$SSH_KEY" ]; then
+    echo ">>> loading ssh key and kown_hosts for default user pontoon..."
+    mkdir ~/.ssh
+    chmod 700 ~/.ssh
+
+    # To preserve newlines, the env var is base64 encoded. Flip it back.
+    echo $SSH_KEY | base64 -d > ~/.ssh/id_rsa
+    echo $KNOWN_HOSTS | base64 -d > ~/.ssh/known_hosts
+    echo $SSH_CONFIG | base64 -d > ~/.ssh/config
+
+    chmod 400 ~/.ssh/id_rsa ~/.ssh/known_hosts 
+    chown -R $(whoami):$(whoami) ~/.ssh/
+    echo "...done."
+fi
+
+echo ">>> Setting up the db for Django"
+python manage.py migrate
+
+# syncing projects if env SYNC_INTERVAL is set, if it is set and you need to "work" on the bash, kill the process syncprojects.sh
+echo ">>> starting continuous syncing projects"
+/app/docker.production/syncprojects.sh &
+
+echo ">>> Starting local server"
+python manage.py runserver 0.0.0.0:8000
```

Comment:
Please make sure to not use django's runserver in production. It's dangerous.

See `Procfile` for what we're using right now as production server.

That might also solve the performance issues you encountered.

Reply:
I'll test performances again